src/lib/eventStack/EventTarget.tsx

```diff
@@ -1,9 +1,10 @@
 import * as _ from 'lodash'
+import { OneOrArray } from 'types/utils'
 
 export default class EventTarget {
-  _handlers = {}
-  _pools = {}
-  target: any
+  private _handlers: { [key: string]: EventListener } = {}
```

Comment:
Heads up, the EventStack will be ported from https://github.com/layershifter/event-stack.  Stardust changes are not likely to survive.  The other module is more up to date and includes bug fixes.

Reply:
I'm aware, thanks, still needed to fix type errors